### DISCUSSION TITLE
Semantic Highlighting in Neovim

### DISCUSSION THREAD

**Discussion by tmwatchanan** [mtshiba/pylyzer#18] [Q&A]
I've seen that `pylyzer` works well for semantic highlighting in VS Code. But how can I enable it in Neovim as well?

**Comment by dstromberg** [comment]
Do linters/type checkers provide the semantics behind syntax highlighting?  I kinda thought that was treesitter's job.

**Reply by tmwatchanan** [reply]
AFAIK pylyzer is considered a language server. In this feature list, I also see it implemented semantic tokens. But it seems lack of documentation how to do so.

**Comment by dheeraj7866** [comment]
how to integrate pylyzer into ci
